src/System Application/App/Language/src/Language.Codeunit.al

```diff
@@ -217,6 +217,48 @@ codeunit 43 Language
         LanguageImpl.SetPreferredLanguageID(UserSecID, NewLanguageID);
     end;
 
+    /// <summary>
+    /// Retrieves the two-letter ISO language name for the specified language ID.
+    /// </summary>
+    /// <param name="LanguageID">The language ID.</param>
+    /// <returns>The two-letter ISO language name.</returns>
+    procedure GetTwoLetterISOLanguageName(LanguageID: Integer): Text
```

Comment:
return Text[2] also would be nice with examples, like "en, us, da, gb" or whatever format is correct. That's a nice way to confirm this returns the format you expect.

Reply:
Next PR